CHANGELOG.md

```diff
@@ -9,6 +9,35 @@ stages](https://github.com/kubernetes/community/blob/master/contributors/devel/s
 
 ## Unreleased
 
+## 0.3.0 - 2019-01-23
+
+### Added
+
+- Add documentation for OVS pipeline.
+- Add Network Policy information to monitoring CRDs for both the Agent and the Controller.
+- Add support for the IPsec ESP protocol for GRE tunnels only; it can be enabled by applying antrea-ipsec.yml instead of antrea.yml.
+- Add framework to develop CLI commands for Antrea; the antctl binary only supports the "version" command at the moment.
+- Add OpenFlow and OVSDB connection health information to the Agent's monitoring CRD.
+- Add octant/octant-antrea-ubuntu Docker image to dockerhub for easier deployment of [Octant] with the Antrea plugin.
+
+### Changed
+
+- Change API group namings (for [CRDs] and Network Policies) from "crd.antrea.io" to "antrea.tanzu.vmware.com" and from "networkpolicy.antrea.io" to "networking.antrea.tanzu.vmware.com".
+- Update to "Antrea on Kind" documentation to indicate that macOS hosts are also supported.
```

Comment:
Is it too small to mention (I just wonder if we able to catch all such changes in every release)?

Reply:
I wanted to keep it around, if "Antrea on Kind" is shown at TGIK it's likely to be on macOS. I wanted to avoid confusion in case someone looked at an older version of the documentation.